src/engraving/draw/color.cpp

```diff
@@ -18,7 +18,7 @@ static void insertHexComponent(int num, std::stringstream& ss);
 
 const Color Color::black { 0, 0, 0 };
 const Color Color::white { 255, 255, 255 };
-const Color Color::transparent { 0, 0, 0, 0 };
+const Color Color::transparent { 255, 255, 255, 0 };
```

Comment:
It seems this change, from transparent black to transparente white, breaks the mtests?

Reply:
See https://github.com/musescore/MuseScore/pull/8928/checks?check_run_id=3420975345#step:8:787

Reply:
strangly only for tuplets though?

Reply:
No, it seems to nt the the `Color::black` lacking the trailing `, 255`? Still, why only for tuplets?
It now wants `<color r="0" g="0" b="0" a="255"/>` being set explicitly.

Reply:
But even if I add it, the tests are still failing...

Reply:
Yeah, something very strange is goind on there. if it were the missinf 255, it should fail at much more places than just the tuplets